bridgetown-core/lib/bridgetown-core/frontmatter_defaults.rb

```diff
@@ -13,7 +13,7 @@ def initialize(site)
     end
 
     def reset
-      @glob_cache = {} if @glob_cache
+      @glob_cache = {} if defined?(@glob_cache)
```

Comment:
With a second look at this, I don't think it needs the `if` statement at all actually.